iocore/net/quic/QUICFrame.h

```diff
@@ -62,17 +61,19 @@ class QUICFrame
   virtual size_t size() const = 0;
   virtual bool is_probing_frame() const;
   virtual size_t store(uint8_t *buf, size_t *len, size_t limit) const = 0;
-  virtual void reset(const uint8_t *buf, size_t len);
+  virtual void reset(){};
```

Comment:
I think we don't need `reset()` anymore because we have `parse()`. `parse()` should reset all fields so we can have it internally (as a private function), but it doesn't need to be a public function I think.